src/render.rs

```diff
@@ -224,19 +262,30 @@ mod tests {
 
     #[test]
     fn relative_links() {
-        // The commented out behaviour is desirable, but not possible
-        // with what ammonia currently offers.
-
-        // let absolute = "[hi](/hi)";
+        let absolute = "[hi](/hi)";
         let relative = "[there](there)";
 
-        for url in &["https://github.com/rust-lang/test",
-                     "https://github.com/rust-lang/test/"] {
-            // let result = markdown_to_html(absolute, Some(url)).unwrap();
-            // assert_eq!(result, "<p><a href=\"https://github.com/rust-lang/test/blob/master/hi\" rel=\"nofollow noopener noreferrer\">hi</a>");
+        for url in &[
+            "https://github.com/rust-lang/test",
+            "https://github.com/rust-lang/test/",
```

Comment:
Should we have gitlab.com & bitbucket.org tests in here to avoid regressions ?

Reply:
Yes, let's.